src/schema.graphql

```diff
@@ -43,6 +44,10 @@ type Mutation {
   createOrganization(nameEn: String!, nameFr: String!, acronymEn: String!, acronymFr: String!): Organization!
   modifyOrganization(id: ID!, data: ModifyOrganizationInput): Organization!
   deleteOrganization(id: ID!): Boolean!
+  createApproval(gcIDApprover: gcIDProfileInput!, gcIDSubmitter: gcIDProfileInput!, requestedChange: RequestedChangeInput, createdOn: String, actionedOn: String, deniedComment: String, status: Status, changeType: ChangeType): Approval!
+  modifyApproval(id: ID!, data: ModifyApprovalInput): Approval!  
+  deleteApproval(id: ID!): Boolean!
```

Comment:
Can remove declaration of the delete mutation.  Deleting can be handled internally